backend/app/agents/devrel/github/tools/github_support.py

```
@@ -0,0 +1,115 @@
+
+OWNER_REPO_RE = re.compile(
+    r'\b([A-Za-z0-9](?:-?[A-Za-z0-9]){0,38})/([A-Za-z0-9._-]{1,100})\b'
+)
+
+REPO_NAME_RE = re.compile(
+    r'\b(?:repo\s+([A-Za-z0-9._-]+)|([A-Za-z0-9._-]+)\s+repo)\b',
+    re.IGNORECASE,
+)
+
+
+async def handle_github_supp(query: str, org: str = None):
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)